### DISCUSSION TITLE
GitHub Achievements

### DISCUSSION THREAD

**Discussion by rhinoxD** [community/community#41093] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I was testing something by creating a repo and deleted it afterwards. Later this evening I saw a quickdraw achievement in my profile and it says 'inaccessible closed within 5 minutes of opening' so I was wondering if there's a way to remove it from my profile as I want to 'achieve' it not by mistake though.

![image](https://user-images.githubusercontent.com/86461976/206457350-61b253e5-87ca-492b-9fa7-8ab038f93b05.png)

**Comment by Anmol-Baranwal** [comment]
You can check my profile [here](https://github.com/Anmol-Baranwal) & you can see that my co-authored commit was counted in a private repository, of which I deleted and the link became inaccessible.

Conclusion: Don't try to test the things in the private testing repository. It will count and will be inaccessible if you delete the repository. Since it is due to the private repository, it will not have much weight.

There is no way to reverse it, although you can remove the display of achievements on your profile but you cannot reverse it.

**Reply by Anmol-Baranwal** [reply]
@rhinoxD  
please mark this as an answer if you have no further queries so that others can know that the question has been answered.

**Comment by DSDmark** [comment, marked as answer]
Hi @rhinoxD,

You can't remove any badge once you're achieved. You can only hide or disable these badge on your profile.

For disable these badges on your profile. Go to your settings and scroll down and uncheck that box and update it.

Here is quick link:- https://github.com/settings/profile

![image](https://user-images.githubusercontent.com/85744816/206843081-d5faee86-3e82-4c18-979a-2728ead7a853.png)

## Or if you want to hide particular badges on your profile, then.

Go to your profile and click on the badge which you want to hide, and then you see that icon on that badge. Click on that. Now your is badge hide. None see your badge on your profile anymore.

![image](https://user-images.githubusercontent.com/85744816/206843165-9002ceab-87fb-449f-b026-b74a2baa13b4.png)

**Comment by chris-short** [comment]
As a community leader, it would be wonderful to be able to create unique Achievement badges for contributors in my orgs.
